packages/botonic-cli/templates/childs/webpack-entries/dev-entry.js

```diff
@@ -0,0 +1,19 @@
+import { DevApp } from '@botonic/react'
```

Comment:
I would leave all templates as they are except the ones about webchat & webviews for several reasons:
1) minimize changes
2) to make examples simpler, to avoid scaring developers if templates are too complex 
3) in case we refactor botonic in the future, it will be easier to modify 1 file per bot rather than 4 files per bot

Reply:
We were talking about this with @marcrabat yesterday, and we finally decided to go this way because:
1. It is a sure and simple way to enforce tree shaking of each artifact.
2. We avoid the "magic" of webpack aliases, I think that's more confusing for the developer than having some extra files with a clear purpose. It provides clarity about how things are built and also provides flexibility in case they want to modify the entry points.
3. This is actually more future proof than the old version, as we "eject" the entry files to the project code, we no longer need to keep that file in botonic source code, so we don't need to worry about backwards compatibility (that responsibility is no longer in botonic). Developers will be able to upgrade botonic versions in their old project without worrying about having to change their webpack config.

Reply:
I agree with separating the entry files, but I was suggesting that for the templates which have no webchat nor webviews, it was a bit confusing having so many files.

Reply:
I think that it's better to have all of them already set up in case the developer wants to add webviews or webchat later (which can be very common).  Otherwise the develeper should need to research these files somewhere and I think won't be favourable for his DX. I understand it can be more cumbersome to maintain, but all the files in `webpack-entries` should be the same for all templates and the entire folder is easy to replicate.